### PR TITLE
[NEW] LC0976: init

### PR DESCRIPTION
**Pull request by novayo**
Question: https://leetcode.com/problems/largest-perimeter-triangle/

---

try_1.cpp: O(nlogn) O(1)

* Runtime: 95 ms, faster than 15.52% of C++ online submissions for Largest Perimeter Triangle.
* Memory Usage: 22 MB, less than 22.88% of C++ online submissions for Largest Perimeter Triangle.

> sort

### CODE REVIEW

File: 0976_Largest_Perimeter_Triangle/try_1.cpp [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Solution {
+public:
+    int largestPerimeter(vector<int>& nums) {
+        sort(nums.begin(), nums.end());
```
Comment (Yabadado) [inline 0976_Largest_Perimeter_Triangle/try_1.cpp:4]: <只是提醒>也可以使用這樣的方式從大到小排列
```suggestion
        sort(nums.rbegin(), nums.rend());
```

Comment (novayo) [reply 0976_Largest_Perimeter_Triangle/try_1.cpp:4]: 這是decending order對吧?
https://www.geeksforgeeks.org/vector-rbegin-and-rend-function-in-c-stl/

Comment (Yabadado) [reply 0976_Largest_Perimeter_Triangle/try_1.cpp:4]: 對的 
C++的STL container我都是到官方document看的
像是vector -> https://cplusplus.com/reference/vector/vector/?kw=vector
裡面會把所有member function都列出來

**Review by Yabadado — APPROVED** [review]
Awesome

**Pull request merged**